composer.json

```diff
@@ -13,11 +13,11 @@
 		"fidry/cpu-core-counter": "^0.5.0",
 		"hoa/compiler": "3.17.08.08",
 		"hoa/exception": "^1.0",
+		"hoa/file": "1.17.07.11",
 		"hoa/regex": "1.17.01.13",
 		"jetbrains/phpstorm-stubs": "dev-master#8ea3bf44722f76a3a32c89fe769b6144c253f8a6",
 		"nette/bootstrap": "^3.0",
 		"nette/di": "3.1.10",
-		"nette/finder": "^2.5",
```

Comment:
Used only in tests with single usage:
- `tests/PHPStan/Rules/Properties/data/efabrica-latte-bug.php`

So I moved it to require-dev.

Reply:
The tool is wrong here. PHPStan depends on ondrejmirtes/BetterReflection which depends on jetbrains/phpstorm-stubs in `require`. So `jetbrains/phpstorm-stubs` is part of production dependencies of PHPStan.

The line here in `require` in `composer.json` is so that PHPStan gets this newer commit from dev-master as opposed to the one ondrejmirtes/BetterReflection depends on.

Reply:
I see, fixed (moved back where it was).

But such intention cannot be ever detected by any rule, so I ignored it with your explanation there.

Thank you

Reply:
Squashed and rebased, should be mergeable again.